lang/cs/Org.Apache.REEF.Wake/Remote/Impl/StreamingLink.cs

```diff
@@ -23,9 +23,9 @@
 using System.Threading;
```

Comment:
I just noticed that the `Dispose(bool)` method is only called from within the class, and always with `true` as the parameter. It seems safe to remove it.